src/App.tsx

```diff
@@ -1,57 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { DropdownMenu } from './components/DropdownMenu/DropdownMenu';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedUser, setSelectedUser] = useState<Person | null>(null);
 
   return (
     <main className="section">
       <h1 className="title">
-        {`${name} (${born} = ${died})`}
+        {selectedUser
+          ? `${selectedUser?.name} (${selectedUser?.born} - ${selectedUser?.died})`
```

Comment:
Maybe you forgot to redeploy the application?
<img width="575" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/54860641/4e29a1f0-d630-4382-b59d-bb915d14929f">